.github/workflows/e2e.yml

```diff
@@ -27,10 +39,16 @@ jobs:
         with:
           fetch-depth: 0
 
+      - uses: cachix/install-nix-action@V27
```

Comment:
Please add a `name` and some comments on why this is different depending on the runner.